src/main/java/minegame159/meteorclient/systems/commands/PrefixListener.java

```diff
@@ -0,0 +1,28 @@
+package minegame159.meteorclient.systems.commands;
+
+import meteordevelopment.orbit.EventHandler;
+import minegame159.meteorclient.MeteorClient;
+import minegame159.meteorclient.events.meteor.CharTypedEvent;
+import minegame159.meteorclient.systems.config.Config;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.ChatScreen;
+
+public class PrefixListener {
```

Comment:
This really doesnt need to be its own class, just put the listener in the already subscribed MeteorClient class